.travis.yml

```diff
@@ -8,8 +8,9 @@ env:
   - RUBY_GC_HEAP_GROWTH_MAX_SLOTS=300000
   - RUBY_GC_HEAP_INIT_SLOTS=600000
   - RUBY_GC_HEAP_GROWTH_FACTOR=1.25
+  - PARALLEL_TEST_PROCESSORS=2
```

Comment:
Travis allows 1.5 cores.  Using more than 2 in the past leads to some spectacular "Killed" failures.